openlibrary/plugins/upstream/utils.py

```diff
@@ -557,6 +556,28 @@ def get_languages():
         _languages = sorted([web.storage(name=d.name, code=d.code, key=d.key) for d in web.ctx.site.get_many(keys)], key=lambda d: d.name.lower())
     return _languages
 
+
+@public
+def get_author_config():
```

Comment:
Nice! Yeah, if we can the url encoding working, we should be able to use this and make the fetch serverside.

Reply:
done!